macos/FluentUI/FilledTemplateImageView/FilledTemplateImageView.swift

```diff
@@ -17,8 +17,7 @@ open class FilledTemplateImageView: NSImageView {
 	///   - fillMask: the mask image used to draw the fill color.
 	///   - contentTintColor: the color to use for the main image outline.
 	///   - fillColor: the color to use for the image fill. When set to nil or clear, the fill won't be drawn.
-	@objc(initWithImage:fillMask:contentTintColor:fillColor:)
-	public init(
+	@objc public init(
 		with image: NSImage,
```

Comment:
Well, this actually makes the Objc signature like this: **initWith:fillMask:contentTintColor:fillColor**. So it skips over the "image" part. So I think I'd prefer to add back the objc decoration here.